kubeflow/kubeflux/README.md

```diff
@@ -0,0 +1,38 @@
+# KubeFlux
+
+KubeFlux is an integration allowing Kubeflow users to leverage [WeaveWorks&#174; Flux]("https://www.weave.works/oss/flux/") for GitOps. KubeFlux utilizes the [stand-alone]("https://github.com/weaveworks/flux/tree/master/site/standalone") implementation. Being stand-alone, most of the maintenance is manual. If you are looking for a more managed solution, we recommend you look at [Weave Cloud&#174;]("https://www.weave.works/product/cloud/").
```

Comment:
Please use 'Weaveworks' instead of 'WeaveWorks', and link to https://github.com/weaveworks/flux instead of https://www.weave.works/oss/flux.